src/lib/Sendbird.tsx

```diff
@@ -374,6 +373,7 @@ const SendbirdSDK = ({
             showSuggestedRepliesFor: configs.groupChannel.channel.showSuggestedRepliesFor,
             suggestedRepliesDirection: configs.groupChannel.channel.suggestedRepliesDirection,
             enableMarkdownForUserMessage: configs.groupChannel.channel.enableMarkdownForUserMessage,
+            enableReactionsSupergroup: undefined as never,
```

Comment:
If you modify this here, won't you be unable to receive changes from the remote as well?

Reply:
Oh I refer to this change made in RN https://github.com/sendbird/sendbird-uikit-react-native/pull/183/files#diff-c566ff50cbbbf3f800ea001b2af36833f1b5e8bcb484b1ccd2a8237345beeee1R215 saw it's set as undefined. Do you think we need to keep as is ? 
```
enableReactionsSupergroup: sdkInitialized && configsWithAppAttr(sdk).groupChannel.channel.enableReactionsSupergroup as never,
```

Reply:
That code is the value injected into UIKitConfigContextProvider, and the code you modified maps the value retrieved from UIKitConfigContext to SendbirdProvider

Reply:
Oh I got confused. Just updated :)